syntax = "proto2";

package sailbot.msg.testing;

option cc_enable_arenas = true;

message EveryTimeMessage {
  optional EveryTimeMessage submsg = 1;
  optional double double_ = 2;
  optional float float_ = 3;
  optional int32 int32_ = 4;
  optional int64 int64_ = 5;
  optional uint32 uint32_ = 6;
  optional uint64 uint64_ = 7;
  optional sint32 sint32_ = 8;
  optional sint64 sint64_ = 9;
  optional fixed32 fixed32_ = 10;
  optional fixed64 fixed64_ = 11;
  optional bool bool_ = 12;
  optional string string_ = 13;
  optional bytes bytes_ = 14;
}
